provider/data_rescue_image_test.go

```diff
@@ -52,3 +52,38 @@ func TestDataRescueImage(t *testing.T) {
 		},
 	})
 }
+
+func TestDataRescueImageAcceptance(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		Providers:  testAccProviders,
+		IsUnitTest: false,
```

Comment:
nit: Is this needed?

Reply:
I was doubting the same but kept it as a verbose indication that this is definitely not a unit test like the one above

Reply:
Fair enough